mne/io/eeglab/eeglab.py

```diff
@@ -510,7 +510,8 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
                 if isinstance(ep.eventtype, int):
                     ep.eventtype = str(ep.eventtype)
```

Comment:
Hm ... can't we lose this line now?

Reply:
if I understand correctly `eventtype` can be an array or even just a single element. This condition checks for the case of a single element